plugins/experimental/memcache/tsmemcache.cc

```diff
@@ -503,7 +503,7 @@ MC::binary_get_event(int event, void *data)
     key         = binary_get_key(this);
     header.nkey = binary_header.request.keylen;
     return get_item();
-  } else if (event == CACHE_EVENT_OPEN_READ_FAILED) {
+  } else if (event == CACHE_EVENT_OPEN_READ_FAILED) { // lgtm[cpp/constant-comparison]
```

Comment:
Assuming `CACHE_EVENT_OPEN_READ_FAILED != TSMEMCACHE_EVENT_GOT_ITEM`, this code is unreachable.

Reply:
That's right! These `else if` blocks are dead code.